frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -161,13 +175,12 @@ function toggleCellTypeIdToDelete(
   };
 }
 
-function deleteByIds<Item>(collection: Item[], collectionToDelete: Item[]) {
+function deleteByItems<Item>(collection: Item[], collectionToDelete: Item[]) {
   return collection.filter((item) => !collectionToDelete.includes(item));
 }
 
 function resetGenesToDeleteAndCellTypeIdsToDelete(
   state: State,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   _: PayloadAction<null>
```

Comment:
I see now!

Reply:
Surprised this isn't already in the rule, is there a thread/issue on this?

Reply:
Yeah I checked eslint doc and they want us to ignore `^_` via the config 🤔 

https://eslint.org/docs/rules/no-unused-vars#argsignorepattern